GetJobber/atlantis
# build(components-native): Forbid html elements in components-native

## Motivations

It is easy to accidentally use html elements in `@jobber/components-native` which do not work. This PR adds an eslint rule to prevent this

## Changes



### Added

- Added eslint rule to `@jobber/components-native` to prevent accidentally using html elements in React Native

### Changed

- 

### Deprecated

- 

### Removed

- 

### Fixed

- 

### Security

- 

## Testing

- Verify disallowed elements can't be used in `packages/components-native`
- Verify disallowed elements can be used in `packages/components`
- Verify disallowed elements can be used in `docs/` storybook files

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

![Random photo of Atlantis](https://loremflickr.com/672/400/atlantis)
